.gitattributes

```diff
@@ -0,0 +1,11 @@
+.appveyor.yml export-ignore
+.editorconfig export-ignore
+.composer export-ignore
+.gitattributes export-ignore
+.github export-ignore
+.gitignore export-ignore
+.php_cs.dist export-ignore
+.travis.yml export-ignore
+build export-ignore
+phpunit export-ignore
+phpunit.xml.dist export-ignore
```

Comment:
I would remove the main `.gitattributes` (root one) as we don't want people to install symfony/symfony anyway. So, there is nothing to do.

Reply:
Thank you.

Reply:
well, depend if we do that in the 4.4 branch only or also in 3.4

Reply:
This PR is for 4.4 only. We cannot remove files (even if nobody should rely on them) in a patch release, that would be rude ;)

Reply:
that can only be for 4.4 to me